usrp_drivers/usrp_driver.cpp

```diff
@@ -309,6 +310,19 @@ void transmit(zmq::context_t &driver_c, USRP &usrp_d, const DriverOptions &drive
               ); //pulse timeit macro
             }
 
+            // Read AGC and Low Power signals
+            usrp_d.clear_command_time();
+            auto read_time = sequence_start_time + uhd::time_spec_t(time_to_send_samples.back()/1.0e6);
```

Comment:
I think if I remember Kev correctly, this read needs to occur right on the tail edge of the pulse. So you would just have to add the pulse length to this read time. Double check with him on that though

Reply:
It would be good to have a config option for this read time I think, since this will depend upon hardware and may be different at different sites. Can you make an agc_signal_read_delay_us config setting, for example?